schemas/2.0.0.json

```diff
@@ -362,7 +362,6 @@
           "$ref": "http://json-schema.org/draft-07/schema#"
         },
         {
-          "type": "object",
```

Comment:
I'd not remove this line but instead, I'd do the following:

```json
"schema": { // Current line 359
  "oneOf": [
    { "type": "boolean" },
    { "allOf": [ // Current line 360
        ...
```

Reply:
It's one the solution, but when we will remove the `type` from second schema, then we infer the type from `http://json-schema.org/draft-07/schema` and it is:

```json
"type": ["object", "boolean"],
```

Reply:
I think that `oneOf` can only complicated (already complex) our JSON Schema

Reply:
You're right. Forgot that `type` can take an array. Go for it then.

Reply:
Fore sure not removing the `type: object` line because that would mean we accept anything and that's not true.

Reply:
But the `"type": ["object", "boolean"]` is inferred from the `http://json-schema.org/draft-07/schema` `allOf` combined schema means that for every sub schema input data must be valid, so if I removed the `type` from the second schema then I have allowed types for first schema `object` and `boolean and for second I have allowed of course everything, but the, as I wrote, the http://json-schema.org/draft-07/schema schema only allows the [object, boolean], so at the end if someone put as input the number type it will be invalid:
- first schema - invalid
- second schema - valid
If someone put boolean or type it will be valid always :)

You can test it with this simple schema:

```
{
  allOf: [
    {
      type: ['object', 'boolean'] 
    },
    {}
  ],
}
```

in the https://www.jsonschemavalidator.net/

Reply:
But for you I will write for second schema the `object, boolean` type 😘

Reply:
Ohh sorry, I didn't notice. Nevermind, it's fine as it is 👍